.devcontainer/cpp/apt-requirements-clang.json

```diff
@@ -6,5 +6,6 @@
   "clang-tools-17": "1:17.0.6~++20240501091250+6009708b4367-1~exp1~20240501211359.8",
   "libclang-rt-17-dev": "1:17.0.6~++20240501091250+6009708b4367-1~exp1~20240501211359.8",
   "lld-17": "1:17.0.6~++20240501091250+6009708b4367-1~exp1~20240501211359.8",
-  "llvm-17": "1:17.0.6~++20240501091250+6009708b4367-1~exp1~20240501211359.8"
+  "llvm-17": "1:17.0.6~++20240501091250+6009708b4367-1~exp1~20240501211359.8",
+  "libyaml-cpp-dev": "0.8.0+dfsg-6build1"
```

Comment:
Don't include -dev dependencies in the final image.

This seems only necessary to compile clang-uml, so add and remove it in the same layer you compile clang-uml in.

Reply:
Turns out it's also required for runtime :( I'll have to keep it here.

Reply:
I don't think so, typically the -dev package has a dependency on the runtime package.